test/apiv2/python/rest_api/test_v2_0_0_container.py

```diff
@@ -97,6 +97,16 @@ def test_attach(self):
     def test_logs(self):
         r = requests.get(self.uri(self.resolve_container("/containers/{}/logs?stdout=true")))
         self.assertEqual(r.status_code, 200, r.text)
+        r = requests.post(
+            self.podman_url + "/v1.40/containers/create?name=topcontainer",
+            json={"Cmd": ["top"], "Image": "alpine:latest"},
+        )
+        self.assertEqual(r.status_code, 201, r.text)
+        payload = r.json()
+        container_id = payload["Id"]
+        self.assertIsNotNone(container_id)
+        r = requests.get(self.podman_url + f"/v1.40/containers/{payload['Id']}/logs?follow=false&stdout=true&until=0")
```

Comment:
Can you set this up to actually test a log file with entries before and after the until?

Reply:
I think what I just changed gets the job done. I call `logs` twice once with `until=0` and another time with `until=1` (arbitrary small value) and compare the length of the logs asserting that 0 should be longer showing that it is working. I am unsure of how to capture and compare the log output more than just the length.

Reply:
actually after messing around with it today I do not think we can parse the logs in these tests since they aren't in the response.